examples/flutter_gallery/lib/demo/material/slider_demo.dart

```diff
@@ -159,6 +159,50 @@ class _SliderDemoState extends State<SliderDemo> {
                 const Text('Continuous'),
               ],
             ),
+            Column(
+              mainAxisSize: MainAxisSize.min,
+              children: <Widget>[
+                Row(
+                  children: <Widget>[
+                    Expanded(
+                      child: Slider(
+                          value: _value,
+                          min: 0.0,
+                          max: 100.0,
+                          onChanged: (double value) {
+                            setState(() {
+                              _value = value;
+                            });
+                          }),
+                    ),
+                    Semantics(
+                      label: 'Editable numerical value',
+                      child: Container(
+                        width: 48,
+                        height: 48,
+                        child: TextField(
+                          onSubmitted: (String value) {
+                            final double myDouble = double.tryParse(value);
+                            if (myDouble != null &&
```

Comment:
the boundaries checked can be removed, then use .clamp()

Reply:
Fixed!